source/OptimaJet.DWKit.StarterApplication/Startup.cs

```diff
@@ -37,21 +39,36 @@ public void ConfigureServices(IServiceCollection services)
         {
             // Add framework services.
             services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
+
             services.AddReact();
+
+            // JWT Auth disabled for now, because we need to 
+            // 1. Add an Auth0 Id column to the users table
+            // 2. Set the DWKitRuntime.Security.CurrentUser
+            // 3. Controller actions are not allowed to have multiple authentication schemes
+            // 4. Cookies must be removed.
+            //
+            // services.AddAuthentication(options =>
+            // {
+            //     options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
+            //     options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
+            // }).AddJwtBearer(options =>
+            // {
+            //     options.Authority = GetVarOrThrow("AUTH0_DOMAIN");
+            //     options.Audience = GetVarOrThrow("AUTH0_AUDIENCE");
+            //     options.RequireHttpsMetadata = false;
+            // });
+
+            // Authentication handeled by auth0
+            // no cookies needed, as auth0 / jwt auth is state-less
             services.AddAuthentication(CookieAuthenticationDefaults.AuthenticationScheme)
-            .AddCookie(options => {
-		options.ExpireTimeSpan = TimeSpan.FromDays(365);
-                options.LoginPath = "/Account/Login/";
+               .AddCookie(options => {
+                    options.ExpireTimeSpan = TimeSpan.FromDays(365);
+                    options.LoginPath = "/Account/Login/";
             });
 
             services.AddMvc(options => {
                 options.Filters.Add(typeof(Security.AuthorizationFilter));
-                options.Filters.Add(

```

Comment:
this type of cache isn't valid with the upgrade -- which is fine, because we don't want responses cached anyway